fl4health/clients/fenda_client.py

```diff
@@ -172,18 +226,38 @@ def get_perfcl_loss(
         aggregated lobal features (z_g) as negative pairs.
         """
 
-        contrastive_loss_minimize = self.compute_contrastive_loss(
+        global_contrastive_loss = self.contrastive_loss(
             features=global_features,  # (z_s)
-            positive_pairs=aggregated_global_features,  # (z_g)
-            negative_pairs=old_global_features,  # (\hat{z_s})
+            positive_pairs=aggregated_global_features.unsqueeze(0),  # (z_g)
+            negative_pairs=old_global_features.unsqueeze(0),  # (\hat{z_s})
         )
-        contrastive_loss_maximize = self.compute_contrastive_loss(
+        local_contrastive_loss = self.contrastive_loss(
             features=local_features,  # (z_p)
-            positive_pairs=old_local_features,  # (\hat{z_p})
-            negative_pairs=aggregated_global_features,  # (z_g)
+            positive_pairs=old_local_features.unsqueeze(0),  # (\hat{z_p})
+            negative_pairs=aggregated_global_features.unsqueeze(0),  # (z_g)
         )
 
-        return contrastive_loss_minimize, contrastive_loss_maximize
+        return global_contrastive_loss, local_contrastive_loss
+
+    def get_mkmmd_loss(
+        self,
+        local_features: torch.Tensor,
+        global_features: torch.Tensor,
+        aggregated_global_features: torch.Tensor,
+    ) -> Tuple[Optional[torch.Tensor], Optional[torch.Tensor]]:
+        """
+        MK-MMD loss aims to compute the distance among given two distributions with optimized betas.
```

Comment:
This comment could be slightly more informative (i.e. the betas are optimized to maximize the statistical power of the measurement.) We're also missing the doc-strings for the args an return values.